data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_12.xml.ep

```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <reg_code><%= $get_var->('SCC_REGCODE') %></reg_code>
+    <install_updates config:type="boolean">true</install_updates>
+    % if (keys %$addons) {
```

Comment:
What could happen if adding non-existent addon ? Installation just continue with a warning ?

Reply:
Installation would fail with err: add-on is not available for registration